google/cloud/bigtable/examples/stock_backtesting/data/AMZN_historical_price.csv

```diff
@@ -0,0 +1,1259 @@
+Date,Open,High,Low,Close,Adj Close,Volume
```

Comment:
I am nor comfortable merging these files.  Can we just include instructions on how to download the files from kaggle?